src/dataDisplay/personPanel/personPanelDetails.jsx

```diff
@@ -196,10 +196,7 @@ const defaultProps = {
     isExpanded: false,
     isMobile: false,
     otherDataGroups: null,
-    selectButtonProps: {
-        label: 'Select',
-        disable: false,
-    },
+    selectButtonProps: {},
```

Comment:
Is this change necessary?  I think we should keep it if possible to avoid regressions.  i.e. by default this button should show and have these props.  And someone (like us) that wishes to suppress this button should have to do something.

Maybe introduce a new `showSelectButton` prop (Boolean, optional, defaults to `true`, and we would set it to `false`).

Reply:
@groberts314 Make sense if it's to avoid possible regressions, but it can be handled from UI as well. Should I add boolean `showSelectButton` and make those changes? Guys, what are your views? @War777 @cbrewer1689

Reply:
I believe this change seems good. Having the default be `{}` makes sense here, because below, in the render, the label's value is being set to "Select" by default and `disable` is already be set to `false` inside of the PersonPanelDetailsActionButton component.

Reply:
So, it sounds like this is fine (change to default value for `selectButtonProps`), but let's also add `showSelectButton` prop as described to govern whether or not to render the button.

^^^ @fantaJinMode

Reply:
If I am thinking correctly `showSelectButton` prop for `<PersonPanelDetails>` component, Yes ??
If that's the case, should I add another prop `showViewRecordButton` for another button as well ? @groberts314 @cbrewer1689

Reply:
`showSelectButton` Should be prop for <PersonPanelDetails> ??
It would be something like this 
```
selectButtonProps: {
      showSelectButton: false,
  },
```
@groberts314 @cbrewer1689

Reply:
@fantaJinMode I'd say yes.  Please default the props to `true` so that by default nothing changes, and people can suppress the buttons by setting new props to `false`.

Reply:
@groberts314 @cbrewer1689 Fixed. Please check once

Reply:
@fantaJinMode @cbrewer1689 Should we restore the prior default value for `selectButtonProps` or should we leave it as an empty object?

Reply:
I think if we're checking `!isEmpty(selectButtonProps)` below we want to restore the default value so that by default, the Select button is displayed, to avoid regressions.  Does that sound right?  Or am I missing something?

Reply:
For avoiding regressions, I agree with @groberts314 of restoring the default value. Please provide your suggestion on this @cbrewer1689. According to that, I will make new changes

Reply:
I think either way is good with me. I don't believe there would be any regressions with how this line current looks.

Reply:
@fantaJinMode I've confirmed with @cbrewer1689 and _really do_ need to revert this change on the default value for `selectButtonProps`.  Because otherwise, callers that are not setting this prop (intending to accept the default value) will have their buttons mysteriously disappear! (because of the `!isEmpty(selectButtonProps)` check).

Reply:
Looking good as of 2ae8cb3.  I consider this resolved, but since we all participated, I'll keep the thread visible for now.